jaxws-ri/tools/wscompile-jdk9/src/main/java/com/sun/tools/ws/ant/WsGen.java

```diff
@@ -45,6 +45,8 @@
 import org.apache.tools.ant.Project;
 import org.apache.tools.ant.types.Path;
 
+import java.lang.module.ModuleFinder;
```

Comment:
java.lang.module.ModuleFinder is a JDK9 API.  WLS needs to continue to run on JDK8.

Reply:
I think that you need to use reflection.  You should add a single method that can be used everywhere that ModuleFinder is introduced in this change.